website/views.py

```diff
@@ -191,23 +192,27 @@ def get_dashboard_nodes(auth):
     """
     user = auth.user
 
-    contributed = user.node__contributed  # nodes user contributed to
-
-    nodes = contributed.find(
-        Q('category', 'eq', 'project') &
-        Q('is_deleted', 'eq', False) &
-        Q('is_registration', 'eq', False) &
-        Q('is_folder', 'eq', False)
+    nodes = Node.find_for_user(
+        user,
+        subquery=(
+            Q('category', 'eq', 'project') &
```

Comment:
@brianjgeiger @sloria made the change to query for ```Q('parent', 'eq', None)``` instead of ```Q('category', 'eq', 'project)``` and the two dashboard node tests ```test_get_dashboard_nodes``` and ```test_get_dashboard_nodes_shows_components_if_user_is_not_contrib_on_project```  **did** indeed fail, both of them finding too many nodes.

Happy to investigate further if I should continue down this path, just let me know.